src/bundle/ui-dev/src/modules/sub-items/components/table-view/table.view.item.component.js

```diff
@@ -343,26 +366,27 @@ export default class TableViewItemComponent extends PureComponent {
     }
 
     render() {
-        const { item, isSelected } = this.props;
+        const { isSelected, showScrollShadowRight } = this.props;
         const editLabel = Translator.trans(/*@Desc("Edit")*/ 'edit_item_btn.label', {}, 'sub_items');
-        const contentTypeIdentifier = item.content._info.contentType.identifier;
-        const contentTypeIconUrl = eZ.helpers.contentType.getContentTypeIconUrl(contentTypeIdentifier);
+        const actionCellClassName = createCssClassNames({
+            'ibexa-table__cell': true,
+            'c-table-view-item__cell': true,
+            'c-table-view-item__cell--actions': true,
+            'c-table-view__cell--shadow-left': showScrollShadowRight,
```

Comment:
I think this class is wrong

Reply:
I have the same shadow on `c-table-view-item__cell` and `c-table-view__cell` (the second one is applied to header cells). I didn't want to duplicate shadows. Should I duplicate them and create `c-table-view-item__cell--shadow-left` or you would prefer other solution?